src/instructions/proctored_exam/download-instructions/LtiProviderInstructions.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+
+const LtiProviderExamInstructions = ({
+  providerName, supportEmail, supportPhone,
+}) => (
+  <>
+    <p>
+      <FormattedMessage
```

Comment:
Just kept the default copy on this page for now, this might need an update in the future.